helmfile/charts/notify-celery/values.yaml

```diff
@@ -156,7 +143,25 @@ nodes:
       hpaEnabled: false
     service:
       serviceEnabled: true
+  # CELERY FOR SENDING SMS OVER DEDICATED LONG CODE
+  other-sms:
+    name: sms
+    newRelicAppName: "notification-celery-sms-dev"
+    scriptArg: "sh /app/scripts/run_celery_sms.sh"
```

Comment:
we should probably rename that script too, so it's obvious what the usage is cc @sastels

Reply:
yes a bit of renaming with "dedicated" or "throttled" in there would certainly help.